packages/block-library/src/page-list/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { pages as icon } from '@wordpress/icons';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit.js';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: _x( 'Page List', 'block title' ),
+	description: __( 'Display a list of all pages.' ),
+	icon,
+	example: {},
```

Comment:
Feels weird that this is defined as an empty object, but I see other blocks are doing this too, so whatever 😄